### PR TITLE
[FIX] hr_timesheet: fix display of empty allocated time in studio

### PR DESCRIPTION
**Pull request by damrOdoo**
This commit's purpose is to fix the display of the default field when entering studio mode from a task form with no allocated time.

Step to reproduce :
-open project
-open office design
-create new task/select a task with 0 allocated hours -open studio

the display of the field 'allocated time' is partially hidden by the span 0%

Source of the bug:
the widget timesheet_uom_no_toggle has a max width of 7CH, which is too little to allow the full display of the default name.

Solution:
Create and add a new scss class which is only active in studio mode.

Version affected:
saas-16.2 to master

task - 3553101
https://www.odoo.com/web#id=3553101&menu_id=4720&cids=1&action=333&active_id=4105&model=project.task&view_type=form

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by MissingNoShiny — COMMENTED** [review]
LGTM, but I think the name of the class is not ideal. Could you rename it to something more descriptive?

File: addons/hr_timesheet/static/src/scss/timesheets_task_form.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 .o_project_kanban .o_kanban_renderer .oe_kanban_align.badge {
     color: inherit;
 }
+
+.o_web_studio_form_view_editor .o_field_widget.o_web_studio_widget_empty.o_planned_hours_studio_class {
+
```
Comment (auon-odoo) [inline addons/hr_timesheet/static/src/scss/timesheets_task_form.scss:6]:
```suggestion
```

**Review by auon-odoo — APPROVED** [review]
Thank you! :)

**Comment by xavierbol** [comment]
robodoo r+

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**